API/Repositories/Implementations/OAuthClientRepository.cs

```diff
@@ -10,11 +10,7 @@ namespace API.Repositories.Implementations;
 public class OAuthClientRepository(OtrContext context) : RepositoryBase<OAuthClient>(context), IOAuthClientRepository
 {
     private readonly OtrContext _context = context;
-
-    public async Task<bool> SecretInUseAsync(string clientSecret)
-    {
-        return await _context.OAuthClients.AnyAsync(x => x.Secret == clientSecret);
-    }
+    public async Task<bool> SecretInUseAsync(string clientSecret) { return await _context.OAuthClients.AnyAsync(x => x.Secret == clientSecret); }
```

Comment:
Prefer overload of `ExistsAsync(string clientSecret)`.

Reply:
I think I botched this, but I think this works - there is a `SecretInUseAsync` method for the secret. Will implement `ExistsAsync(int clientId, string clientSecret)`.